spec/controllers/admin/videos_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'spec_helper'
+
+describe Admin::VideosController do
+  describe "GET new" do
+    it_behaves_like 'require_sign_in' do
+      let(:action) { get :new }
+    end
+    it_behaves_like 'require_admin' do
+      let(:action) { get :new }
+    end
+    it "sets @video to a new video record with admin" do
+      set_current_admin
+      get :new
+      expect(assigns(:video)).to be_new_record
+      expect(assigns(:video)).to be_instance_of(Video)
+    end
+  end
+
+  describe "POST create" do
+    it_behaves_like 'require_sign_in' do
+      let(:action) { post :create }
+    end
+    it_behaves_like 'require_admin' do
+      let(:action) { post :create }
+    end
+
+    context "with valid input" do
+      before do
+        set_current_admin
+        category = Fabricate(:category)
+        post :create, video: { title: "Paul", description: "A funny alien movie.", categories: category.id}
+      end
+      it "redirects to the add new video page" do
```

Comment:
Reviewed all spec and added a blank line between each examples, this practice improves the readability indeed.